Compilation broken

```
make[2]: Entering directory `/usr/src/RPM/BUILD/zabbix-module-sockets-1.1.0/src/modules/sockets'
/bin/sh ../../../libtool  --tag=CC   --mode=compile x86_64-alt-linux-gcc -DHAVE_CONFIG_H -I. -I../../..     -I/usr/include/zabbix -pipe -Wall -g -O2 -c -o libzbxsockets_la-libzbxsockets.lo `test -f 'libzbxsockets.c' || echo './'`libzbxsockets.c
/bin/sh ../../../libtool  --tag=CC   --mode=compile x86_64-alt-linux-gcc -DHAVE_CONFIG_H -I. -I../../..     -I/usr/include/zabbix -pipe -Wall -g -O2 -c -o libzbxsockets_la-tcp.lo `test -f 'tcp.c' || echo './'`tcp.c
/bin/sh ../../../libtool  --tag=CC   --mode=compile x86_64-alt-linux-gcc -DHAVE_CONFIG_H -I. -I../../..     -I/usr/include/zabbix -pipe -Wall -g -O2 -c -o libzbxsockets_la-unix.lo `test -f 'unix.c' || echo './'`unix.c
libtool: compile:  x86_64-alt-linux-gcc -DHAVE_CONFIG_H -I. -I../../.. -I/usr/include/zabbix -pipe -Wall -g -O2 -c unix.c  -fPIC -DPIC -o .libs/libzbxsockets_la-unix.o
libtool: compile:  x86_64-alt-linux-gcc -DHAVE_CONFIG_H -I. -I../../.. -I/usr/include/zabbix -pipe -Wall -g -O2 -c tcp.c  -fPIC -DPIC -o .libs/libzbxsockets_la-tcp.o
libtool: compile:  x86_64-alt-linux-gcc -DHAVE_CONFIG_H -I. -I../../.. -I/usr/include/zabbix -pipe -Wall -g -O2 -c libzbxsockets.c  -fPIC -DPIC -o .libs/libzbxsockets_la-libzbxsockets.o
In file included from libzbxsockets.h:13:0,
                 from tcp.c:1:
tcp.c: In function 'tcp_stat':
tcp.c:124:63: error: expected ')' before 'path'
   SET_MSG_RESULT(result, zbx_dsprintf("Error reading %s: %s", path, zbx_strerror(errno)));
```

--- a/README.md
+++ b/README.md
@@ -43,7 +43,7 @@ sockets.sendq[protocol,<state>] returns the sum of all buffers in the send queue
 sockets.recvq[protocol,<state>] returns the sum of all buffers in the receive
                                 queue for sockets of the given protocol
 
-sockets.modver[]                version of the loaded module
+sockets.modver                  version of the loaded module
 ```
 
 ### Supported protocols

--- a/configure.ac
+++ b/configure.ac
@@ -51,7 +51,7 @@ echo
 echo "************************************************************"
 echo "*                 Now run '${am_make} install'                   *"
 echo "*                                                          *"
-echo "*            Thank you for using libzbxsystemd!            *"
+echo "*            Thank you for using libzbxsockets!            *"
 echo "*               <http://cavaliercoder.com>                 *"
 echo "************************************************************"
 echo

--- a/src/modules/sockets/tcp.c
+++ b/src/modules/sockets/tcp.c
@@ -121,7 +121,7 @@ int tcp_stat(
 
 	if (ferror(f)) {
 		fclose(f);
-		SET_MSG_RESULT(result, zbx_dsprintf("Error reading %s: %s", path, zbx_strerror(errno)));
+		SET_MSG_RESULT(result, zbx_dsprintf(NULL, "Error reading %s: %s", path, zbx_strerror(errno)));
 		return res;
 	}
 

--- a/src/modules/sockets/unix.c
+++ b/src/modules/sockets/unix.c
@@ -77,7 +77,7 @@ int unix_count(AGENT_REQUEST *request, AGENT_RESULT *result)
 
 	if (ferror(f)) {
 		fclose(f);
-		SET_MSG_RESULT(result, zbx_dsprintf("Error reading " _PATH_PROCNET_UNIX ": %s", zbx_strerror(errno)));
+		SET_MSG_RESULT(result, zbx_dsprintf(NULL, "Error reading " _PATH_PROCNET_UNIX ": %s", zbx_strerror(errno)));
 		return res;
 	}
 

